exercises/alphametics/example.rb

```diff
@@ -0,0 +1,41 @@
+class Alphametics
+  VERSION = 1
+
+  def solve(puzzle)
+    letters = Hash.new(0)
+    puzzle = format_to_ruby(puzzle)
```

Comment:
`as_ruby_expression` seems good. I didn't add close the `eval` because it would be an extra overhead for the loop.